Wootric/slate
# Add section for Email Templates and API

Changes:
1. `Email Templates` which links to http://help.wootric.com/email-surveys
2. `Email API` which links to https://docs.wootric.com/api/#email-survey

Test:
1. Checkout the branch locally
2. Check if we have `Email Templates` and `Email API` section on landing page

Trello: https://trello.com/c/81GAt7YY/3916-we-need-to-add-a-section-about-email-api-templates-triggers-on-our-docs-page-customer-feedback
